packages/client/wallets/aa/src/types/Config.ts

```diff
@@ -43,7 +47,7 @@ export type FireblocksNCWSigner =
           deviceId: string;
       });
 
-type Signer = FireblocksNCWSigner | ethers.Signer | Web3AuthSigner;
+type Signer = FireblocksNCWSigner | EIP1193Provider | Web3AuthSigner;
```

Comment:
is fireblocks still supported? Looks like we dropped it [here](https://github.com/Crossmint/crossmint-sdk/pull/379/files#diff-d4de4d77c0ae39dbb040b4fa7d344b1d1522fdb13ca57007c6cf2b123081770aL36) which is totally fine, let's just remove this option for now if it's not supported